lib/services/batchManagement/package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "azure-arm-batch",
+  "author": "Microsoft Corporation",
+  "contributors": [
+    "Wu, Xing <xingwu@microsoft.com>"
+  ],
+  "version": "0.15.0",
```

Comment:
This is your first version.
https://www.npmjs.com/search?q=azure-arm-batch
It should be "0.1.0"